.dockerignore

```diff
@@ -19,4 +21,12 @@
 !Carog.lock
 !Cargo.toml
 !/docker
+!/target/release/client-cli
+!/target/release/client-rpc
+!/target/release/chain-abci
+!/target/release/dev-utils
+!/target/debug/client-cli
+!/target/debug/client-rpc
+!/target/debug/chain-abci
+!/target/debug/dev-utils
 /docker/Dockerfile
```

Comment:
During integration tests, the working directory should be inside `./integration-tests/`. Seem the built binaries are not exposed to docker here? @tomtau @linfeng-crypto

Reply:
integration-tests uses the launcher,  the reason is some binaries are disappeared, although all the build command run successfully.

Reply:
so they are ignored / not preserved?

Reply:
I mean it seems to need a rule 
‘’’
!/integration-tests/target/debug/client-cli
...
‘’’